enum/LICENSE

```diff
@@ -0,0 +1,32 @@
+Copyright (c) 2013, Ethan Furman.
```

Comment:
I'm gonna add this to the readme. Please do `pip install enum34` which will add enum to your python environment. You won't have to do anything with pulling in enum anymore

You should delete the enum files as a result

Reply:
I think that should fix the Travis failure as well (pushes validate your changes, and you can see results in the checks tab)